diff --git a/GSoC 2016/AES.ipynb b/GSoC 2016/AES.ipynb
index 132db87..08c7713 100644
--- a/GSoC 2016/AES.ipynb	
+++ b/GSoC 2016/AES.ipynb	
@@ -71,6 +71,57 @@
   },
   {
    "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "\"/home/juser/notes/GSoC 2016\""
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "WARNING: Base.writemime is deprecated.\n",
+      "  likely near /opt/julia_packages/.julia/v0.5/IJulia/src/kernel.jl:31\n",
+      "in display_dict at /opt/julia_packages/.julia/v0.5/IJulia/src/execute_request.jl\n"
+     ]
+    }
+   ],
+   "source": []
+  },
+  {
+   "cell_type": "code",
    "execution_count": null,
    "metadata": {
     "collapsed": true